src/server/pfs/cmds/cmds.go

```diff
@@ -1016,6 +1021,66 @@ Any pachctl command that can take a commit, can take a branch name instead.`,
 	shell.RegisterCompletionFunc(deleteBranch, shell.BranchCompletion)
 	commands = append(commands, cmdutil.CreateAliases(deleteBranch, "delete branch", branches))
 
+	FindCommits := &cobra.Command{
+		Use:   "{{alias}} <repo>@<branch-or-commit>:<path/in/pfs> [flags]",
+		Short: "search for commits with reference to <filePath> within a branch starting from <repo@commitID>",
+		Long:  "search for commits with reference to <filePath> within a branch starting from <repo@commitID>",
+		Run: cmdutil.RunFixedArgs(1, func(args []string) error {
+			file, err := cmdutil.ParseFile(project, args[0])
```

Comment:
What do you think about making this another flag?  Since `FindCommit` is general enough to be extended with other matching criteria, we shouldn't assume a single argument as one of those criteria.

Maybe `--modifies`, `--modifies-path, `--changes-path`?

Reply:
I think that change might be best left to do once we introduce other criteria for findCommit

Reply:
That would be a breaking change.  We don't really have strong compatibility guarantees though, so it's probably fine for now.

Reply:
+1 on using a flag for this